src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 2.2.21
 ++++++
 * vm create: Added --computer-name for setting a vm's computer name.
+* vm/vmss create: `--ssh-key-value` can now accept multiple ssh public key values or paths.
```

Comment:
Then shouldn't it be `--ssh-key-values`?  This would not be a breaking change due to the way argparse parses things.

Reply:
Yeah, I didn't want cause confusion, even though it would work. but that concern was probably unnecessary.